lighthouse-core/lib/emulation.js

```diff
@@ -113,9 +120,19 @@ function goOffline(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', OFFLINE_METRICS);
 }
 
+function enableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', CPU_THROTTLE_METRICS);
+}
+
+function disableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', NO_CPU_THROTTLE_METRICS);
+}
+
```

Comment:
We could use https://developer.chrome.com/extensions/system_cpu in the extension and require('os') in the cli and update it later if we know a better solution